Libraries/Components/TimePickerAndroid/TimePickerAndroid.android.js

```diff
@@ -59,7 +58,7 @@ class TimePickerAndroid {
    * being undefined. **Always** check whether the `action` before reading the values.
    */
   static async open(options: TimePickerOptions): Promise<TimePickerResult> {
-    return TimePickerModule.open(options);
+    return NativeTimePickerAndroid.open(options);
```

Comment:
after the first change you will need to do this:
```
if (NativeTimePickerAndroid) {
  return NativeTimePickerAndroid.open(options);
}
```

Reply:
Thanks, I've consolidated both iOS and Android implementations in https://github.com/facebook/react-native/pull/24897/commits/b45ea6179e0f334dcb444bae082d818e4dfa0b88